### DISCUSSION TITLE
Network Topology

### DISCUSSION THREAD

**Discussion by KrystianPlichta** [netbox-community/netbox#6720] [Q&A] [answered]
Is there any way to generate the network topology or are there any plugins to do this?

**Comment by rmrf-run** [comment]
I have had good luck with [this](https://github.com/mattieserver/netbox-topology-views)

A [quick search](https://github.com/search?q=netbox-plugin) turns up a few more

**Comment by julianze** [comment]
A bit more features you will find here:
https://github.com/iDebugAll/nextbox-ui-plugin

**Comment by ryanmerolle** [comment, marked as answer]
old post but most netbox plugins get tagged accordingly and show up in our github topic: https://github.com/topics/netbox-plugin
